api/services/elastigroup/azure/spotVms/paths/vmLogs.yaml

```diff
@@ -10,15 +10,15 @@ get:
     - $ref: "../../../../../commons/parameters/accountId.yaml"
     - $ref: "../../../../../commons/parameters/groupId.yaml"
     - in: query
-      name: FROM_DATE
```

Comment:
@vlad666spot this was addressed in the previous PR which was approved, please make sure to align this branch with master